spec/features/comments/view_comments_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+feature 'Feature: view all comments.' do
+  context 'As an authenticated user' do
+    let(:user) { FactoryGirl.create(:user) }
+
+    before :each do
+      sign_in_as user
+    end
+
+    scenario 'See all comments on the sitcom show page' do
+      review = FactoryGirl.create(:review)
+      FactoryGirl.create(:comment, body: "I hate your review", user: user, review: review)
+      FactoryGirl.create(:comment, body: "I love your review", user: user, review: review)      
```

Comment:
Line is too long. [96/80]<br>Unnecessary spacing detected.<br>Trailing whitespace detected.